wire-ios/Wire-iOS/Sources/Managers/DatadogWrapper.swift

```diff
@@ -150,6 +150,7 @@ extension DatadogWrapper: RemoteLogger {
 // MARK: Crypto helper
 
 import CryptoKit
+import UIKit
```

Comment:
can you put it on the line 29 so inside // MARK: - DATADOG ENABLED
?

Reply:
Done!